src/SiteBuilder.php

```diff
@@ -55,7 +55,7 @@ private function cleanup()
 
     private function buildSite($source, $destination, $siteData)
     {
-        $result = collect($this->files->allFiles($source, true))->map(function ($file) use ($source) {
+        $result = collect($this->files->allFiles($source))->map(function ($file) use ($source) {
```

Comment:
This is `true` because in `SiteBuilder`, we _do_ want to process files that begin with a `.`—`.htaccess` files, for example—so that they get picked up by the `DefaultHandler` and copied. 

However, we also need to ignore `.DS_Store` files so we don't end up copying a bunch of junk to the `build` directory. Previously, that was filtered out in `IgnoredHandler`, but now I've moved that directly into `Filesystem -> allFiles()`, and removed the need for ever having to passing a boolean to `allFiles()`.

Reply:
@damiani ahhh i missed that update to the parameters on `allFiles`. Thanks!

Reply:
that `allFiles` method is full of jank